deployment/nginx/conf/nginx.conf

```diff
@@ -0,0 +1,50 @@
+events {}
+http {
+    server {
+        listen 80 default_server;
+        listen [::]:80 default_server;
+
+        location / {
+            return 301 https://$host$request_uri;
+        }
+    }
+    server {
+        listen 443 ssl http2;
+        listen [::]:443 ssl http2;
+
+        ssl_certificate /etc/letsencrypt/live/blue.bicikl-project.eu-0001/fullchain.pem;
```

Comment:
If you run it somewhere else you will need to changes these variables

Reply:
True, these will need to be set to those of the cetaf server. I think that is the next step in deployment and will probably make a new PR for that and also include the other changes like the .env file you suggested